tests/unit/conftest.py

```diff
@@ -80,3 +81,33 @@ def teardown():
 
     with app.app_context():
         yield app
+
+
+@pytest.yield_fixture(scope='session', autouse=True)
+def db_app(request):
```

Comment:
@jacquerie @mihaibivol I wanted an app context that had tables created only but without the delaying factor of the ES integration. Let me know if you think this is okay, or if it should be moved to `tests/integration`.